spec/controllers/application_controller_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+describe ApplicationController, type: :controller do
```

Comment:
I think a request spec would be better here rather than calling the controller class directly. We never call controller classes directly.

Reply:
ok will convert my concreet controller spec to a request spec :+1:. As for this fiel it is testing the abscract class 'ApplicationController' so might not be able to use a request spec as it is not routable